### ISSUE TITLE
iOS devices unable to play sound in UV version 4.0.22 - issue with HLS streaming (British Library)

### ISSUE THREAD

**Issue by damodeburca** [UniversalViewer/universalviewer#932]
**UV version:**
 universalviewer@4.0.22
```

**I'm submitting a:**
<!-- (check one with "x") -->
- [x] bug report => please fork one of these [codesandbox examples](https://github.com/UniversalViewer/universalviewer/wiki/UV-Examples) with a repro of your issue and include a link to it below
- [ ] feature request => please use the [user stories repo](https://github.com/UniversalViewer/user-stories)
- [ ] support request => Please do not submit support requests here, use [stackoverflow](http://stackoverflow.com/questions/tagged/iiif)

**Current behavior:**
Unable to use iOS devices to play sound - there appears to be an issue with HLS streaming

**Expected behavior:**
Sounds should play - 
We were able to play sounds in UV version 4.0.19
**Steps to reproduce:**
Using any iOS device go to this URL, making sure that you are using uv4.0.22: 
https://access-beta.bl.uk/item/viewer/ark:/81055/vdc_100133103553.0x000002
Error message appears saying no media selected

**Related code:**

```
I believe the recent change to the iiif-av-component might be related to this issue:
https://github.com/IIIF-Commons/iiif-av-component/pull/89
```

**Other information:**
<!-- List any other information that is relevant to your issue. Stack traces, related issues, suggestions on how to fix, Stack Overflow links, forum links, etc. -->

**Comment by damodeburca** [comment]
Hello @hackartisan this is the link to the manifest for this item: https://api.bl.uk/metadata/iiif/ark:/81055/vdc_100133103553.0x000002/manifest.json

**Comment by damodeburca** [comment]
Hello @LlGC-szw, As we briefly discussed on the 28/9 call, I guess from the BL point of view it would be good to have an indication when this issue is likely to be merged to the next UV4 community release. Many thanks.

**Comment by LlGC-szw** [comment]
Flagged to be discussed in the Steering Group.

**Comment by tpendragon** [comment]
We think if https://github.com/IIIF-Commons/iiif-av-component/pull/92 is released and included in UV it will fix this issue.

**Comment by edsilv** [comment]
Have just published v4.0.24 including this fix
